src/main/target/common_post.h

```diff
@@ -26,3 +26,69 @@
 # undef VTX_SMARTAUDIO
 # undef VTX_TRAMP
 #endif
+
+// Forced configuration of two software serials and SERIAL_PORT_COUNT recalc
+#ifndef USE_SOFTSERIAL1
+#  define USE_SOFTSERIAL1
+#endif
```

Comment:
It has been requested by @borisbstyle to have two soft serials configured for all targets.